.gitmodules

```diff
@@ -49,7 +49,7 @@
 	branch = release/2.0.0
 [submodule "src/fsharp"]
 	path = src/fsharp
-	url = https://github.com/Microsoft/VisualFSharp
+	url = https://github.com/dagood/VisualFSharp
```

Comment:
Looks like all the F# changes are in the upstream repo, I don't think we should need to change this?

Reply:
They aren't quite yet. These are the new ones, one from https://github.com/Microsoft/visualfsharp/pull/4674 and a few fixup attempts by me: https://github.com/dagood/VisualFSharp/compare/6417ab1...c5f5c2c2e5a6178e9c260e83605b699b08080996